src/Hazelcast.Net/Networking/NetworkingOptions.cs

```diff
@@ -180,6 +180,9 @@ private NetworkingOptions(NetworkingOptions other)
     /// socket. It is also the timeout for cloud discovery.</para>
     /// </remarks>
     public int ConnectionTimeoutMilliseconds { get; set; } = 5_000;
+    
+    // TODO: Refactor when new networking options are added.
```

Comment:
This work is to introduce the new networking option, why are we introducing this with a TODO to refactor immediately?

Reply:
These are note to my self to handle it on next PRs as explained above.